index.html

```diff
@@ -40,6 +40,32 @@ <h1 class="main-title">
                 Saudi Arabia is rich in heritage and history. The country is home to hundreds of historically important sites.
             </p>
         </div>
+        <div class="info">
+            <div>
+                <div class="info-div">
+                    <h2>
```

Comment:
Does heading and paragraph require not styling?

Reply:
added (styles)